uaa/src/main/webapp/WEB-INF/spring-servlet.xml

```diff
@@ -248,7 +248,13 @@
     </bean>
 
     <bean id="contentSecurityPolicyFilter"
-          class="org.cloudfoundry.identity.uaa.security.web.ContentSecurityPolicyFilter"/>
+          class="org.cloudfoundry.identity.uaa.security.web.ContentSecurityPolicyFilter">
+        <property name="cspAllowedScriptSrc"
+                  value="#{@config['contentSecurityPolicy']==null ? T(java.util.Arrays).asList('self') :
```

Comment:
if you simply set self then CSP does not work. 'self' is wanted - I assume
-> ('''self''')